frontend/packages/data-portal/app/utils/filter.ts

```diff
@@ -0,0 +1,42 @@
+import { isNumber } from 'lodash-es'
+
+import { Runs_Bool_Exp } from 'app/__generated__/graphql'
+import { DEFAULT_TILT_MAX, DEFAULT_TILT_MIN } from 'app/constants/tiltSeries'
```

Comment:
Nit: Could these constants be renamed as `DEFAULT_TILT_RANGE_MAX` and `DEFAULT_TILT_RANGE_MIN`?